test/functional/apps/discover/_discover.ts

```diff
@@ -26,14 +27,29 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     defaultIndex: 'logstash-*',
   };
 
+  const logTypes = (msg: string = '') => async () =>
+    log.debug(
+      `\n### Saved Object Types In Index: [.kibana] ${msg}: \n${inspect(
+        await savedObjectInfo.types(),
+        {
+          compact: false,
+          depth: 99,
+          breakLength: 80,
+          sorted: true,
+        }
+      )}`
+    );
```

Comment:
Do we really need to run this aggregation and log these types every time we run the tests? If so, can't we just put it into `importExport.load()`? 

I don't feel like this is necessary though.